docs/gwcs/user_introduction.rst

```diff
@@ -0,0 +1,462 @@
+.. _user_intro:
+
+Using Generalized World Coordinate System Models
+================================================
+
+Brief Introduction and Outline
+------------------------------
+
+`GWCS (Generalized World Coordinate System) <https://github.com/spacetelescope/gwcs>`__
+package is intended as a generalization of the FITS WCS standards, which basically provides
+a transformation from detector array coordinates to sky, spectral, time or any other physical
```

Comment:
I'm going to suggest a strict difference between "pixel" and "array" coordinates as used by APE 14, where "array" is in numpy (C) order. See [the ndcube docs](https://docs.sunpy.org/projects/ndcube/en/stable/introduction.html#important-concepts-array-pixel-and-world-axes) for how we explain this to our users.

```suggestion
a transformation from detector pixel coordinates to sky, spectral, time or any other physical
```

Reply:
This seems reasonable and this change has been made. However...